pkg/tfgen/docs.go

```diff
@@ -2358,3 +2361,53 @@ var (
 func guessIsHCL(code string) bool {
 	return guessIsHCLPattern.MatchString(code)
 }
+
+func plainDocsParser(docFile *DocFile, pkgName string, g *Generator) ([]byte, error) {
+	// Replace upstream front matter with Pulumi registry's
```

Comment:
Typo?
```suggestion
	// Replace upstream front matter within Pulumi's registry
```

Reply:
would `Replace upstream front matter with Pulumi registry's front matter` be more clear? we're just subbing in what our registry needs here.

Reply:
Yes. Much more clear.